README.rst

```diff
@@ -51,6 +51,14 @@ System requirements:
 - gobject-introspection type libraries for GTK+, Pango
 - GTK+ 3.10 or newer (3.14 or newer for best results)
 
+On Debian/Ubuntu, the following packages may need installation:
+- python-docutils
+- gettext
+- python-pip
+- pkg-config
+- libcairo-dev
+- libgirepository1.0-dev
```

Comment:
These are needed when you're pip installing pygobject and pycairo, instead of satisfying those dependencies with Debian packages.

This list is also missing those gir1.2-* packages.  The 'ppa' branch should [have a list](https://github.com/gtimelog/gtimelog/blob/ppa/debian/control#L20-L27), although I see it's a bit outdated (lists appindicator, which is not required, and forgets keyring, which may come in handy).